Description: Add icons in different sizes
Author: Benjamin Drung <bdrung@ubuntu.com>
Bug: http://jira.atheme.org/browse/AUD-208

--- audacious-2.4~alpha1.orig/pixmaps/Makefile
+++ audacious-2.4~alpha1/pixmaps/Makefile
@@ -3,4 +3,29 @@ DATA = audacious.png \
 
 include ../buildsys.mk
 
-PACKAGE = pixmaps
+install-extra:
+	for size in 16 22 24 32 36; do \
+		i=audacious-$${size}.png; \
+		${INSTALL_STATUS}; \
+		if ${INSTALL} -D -m 644 $$i ${DESTDIR}${datadir}/icons/hicolor/$${size}x$${size}/apps/audacious.png; then \
+			${INSTALL_OK}; \
+		else \
+			${INSTALL_FAILED}; \
+		fi \
+	done
+
+	i=audacious.png; \
+	${INSTALL_STATUS}; \
+	if ${INSTALL} -D -m 644 $$i ${DESTDIR}${datadir}/icons/hicolor/48x48/apps/audacious.png; then \
+		${INSTALL_OK}; \
+	else \
+		${INSTALL_FAILED}; \
+	fi
+
+	i=audacious.svg; \
+	${INSTALL_STATUS}; \
+	if ${INSTALL} -D -m 644 $$i ${DESTDIR}${datadir}/icons/hicolor/scalable/apps/audacious.svg; then \
+		${INSTALL_OK}; \
+	else \
+		${INSTALL_FAILED}; \
+	fi
