Description: Add missing "playlist end reached" hook.
Author: John Lindgren <john.lindgren@tds.net>
Bug-Ubuntu: https://launchpad.net/bugs/625903
Bug: http://jira.atheme.org/browse/AUDPLUG-295
Origin: upstream, http://hg.atheme.org/audacious/audacious/rev/7aad0c1166be

diff -r fc61e92b1258 -r 7aad0c1166be src/audacious/playback.c
--- a/src/audacious/playback.c	Thu Sep 23 20:44:41 2010 -0400
+++ b/src/audacious/playback.c	Sat Sep 25 10:12:00 2010 -0400
@@ -316,6 +316,7 @@
         if (! play)
         {
             complete_stop ();
+            hook_call ("playlist end reached", NULL);
             break;
         }
 

