Description: Make symlinks relative.  Closes: AUD-172.
Author: John Lindgren <john.lindgren@tds.net>
Origin: upstream, http://hg.atheme.org/audacious/audacious/rev/9fbc433ae3fb

diff -r c789415cead1 -r 9fbc433ae3fb src/audacious/Makefile
--- a/src/audacious/Makefile	Sun Apr 11 11:49:04 2010 -0400
+++ b/src/audacious/Makefile	Sun Apr 11 12:05:24 2010 -0400
@@ -176,7 +176,7 @@
 
 install-extra:
 	if test -h "${DESTDIR}${bindir}/audacious" ; then ${RM} "${DESTDIR}${bindir}/audacious" ; fi
-	${LN_S} "${DESTDIR}${bindir}/audacious2" "${DESTDIR}${bindir}/audacious"
+	${LN_S} audacious2 "${DESTDIR}${bindir}/audacious"
 
 	if test x"${PROG_IMPLIB_NEEDED}" = x"yes"; then \
 		for i in ${PROG}; do \
diff -r c789415cead1 -r 9fbc433ae3fb src/audtool/Makefile
--- a/src/audtool/Makefile	Sun Apr 11 11:49:04 2010 -0400
+++ b/src/audtool/Makefile	Sun Apr 11 12:05:24 2010 -0400
@@ -29,7 +29,7 @@
 install-extra:
 	if test -h "${DESTDIR}${bindir}/audtool" ; then ${RM} "${DESTDIR}${bindir}/audtool" ; fi
 	mkdir -p "${DESTDIR}${bindir}"
-	${LN_S} "${DESTDIR}${bindir}/audtool2" "${DESTDIR}${bindir}/audtool"
+	${LN_S} audtool2 "${DESTDIR}${bindir}/audtool"
 
 uninstall-extra:
 	if test -h "${DESTDIR}${bindir}/audtool" ; then ${RM} "${DESTDIR}${bindir}/audtool" ; fi
