Description: Symlink man pages too.
Author: Benjamin Drung <bdrung@ubuntu.com>
Bug: http://jira.atheme.org/browse/AUD-207

--- audacious-2.4~alpha1.orig/man/Makefile
+++ audacious-2.4~alpha1/man/Makefile
@@ -1,3 +1,14 @@
 MAN = audacious2.1 audtool2.1
 
 include ../buildsys.mk
+
+install-extra:
+	${MKDIR_P} "${DESTDIR}${mandir}/${mansubdir}"
+	if test -h "${DESTDIR}${mandir}/${mansubdir}/audacious.1" ; then ${RM} "${DESTDIR}${mandir}/${mansubdir}/audacious.1" ; fi
+	${LN_S} audacious2.1 "${DESTDIR}${mandir}/${mansubdir}/audacious.1"
+	if test -h "${DESTDIR}${mandir}/${mansubdir}/audtool.1" ; then ${RM} "${DESTDIR}${mandir}/${mansubdir}/audtool.1" ; fi
+	${LN_S} audtool2.1 "${DESTDIR}${mandir}/${mansubdir}/audtool.1"
+
+uninstall-extra:
+	if test -h "${DESTDIR}${mandir}/${mansubdir}/audacious.1" ; then ${RM} "${DESTDIR}${mandir}/${mansubdir}/audacious.1" ; fi
+	if test -h "${DESTDIR}${mandir}/${mansubdir}/audtool.1" ; then ${RM} "${DESTDIR}${mandir}/${mansubdir}/audtool.1" ; fi
